app/src/test/kotlin/com/wire/android/mapper/MessageMapperTest.kt

```diff
@@ -150,9 +150,9 @@ class MessageMapperTest {
         )
 
         assert(
```

Comment:
Suggestion: Can we replace this plain assert for one of the libs we are using like `kluent` for example :D 

I know it is not related to this PR, but since we are going through this test. This, when fails, will provide more relevant info